src/AimInputComponent.cpp

```diff
@@ -18,27 +19,36 @@ AimInputComponent::~AimInputComponent() {
 void AimInputComponent::pollInput() {
     if (glfwGetKey(window, GLFW_KEY_I) == GLFW_PRESS) {
         toggleXRotation = true;
-        velocity = 1.0f;
+        rotationDirection = 1.0f;
     } else if (glfwGetKey(window, GLFW_KEY_K) == GLFW_PRESS) {
         toggleXRotation = true;
-        velocity = -1.0f;
+        rotationDirection = -1.0f;
     } else {
         toggleXRotation = false;
     }
 
     if (glfwGetKey(window, GLFW_KEY_J) == GLFW_PRESS) {
         toggleYRotation = true;
-        velocity = 1.0f;
+        rotationDirection = 1.0f;
     } else if (glfwGetKey(window, GLFW_KEY_L) == GLFW_PRESS) {
         toggleYRotation = true;
-        velocity = -1.0f;
+        rotationDirection = -1.0f;
     } else {
         toggleYRotation = false;
     }
 
+    toggleThrow = false;
     if (glfwGetKey(window, GLFW_KEY_SPACE) == GLFW_PRESS) {
-        toggleThrow = true;
+        if(!spacePressed){
```

Comment:
You're totally right on the naming. I changed it, dont wan't to reset pressStart, as I think this might be reused for other things, as it gives you the information at what time space was pressed the last time